src/components/IndexTabs/IndexTab.js

```diff
@@ -34,8 +39,26 @@ const IndexTab = ({ tabs }) => {
             )
           })}
         </div>
+        <div className={styles.actions}>
+          {bottomActions
+            .filter(({ showOnTabs, hide }) => {
+              if (hide) {
+                return 0
+              }
+              if (showOnTabs) {
+                return showOnTabs.includes(title)
+              }
 
-        <SignalModal canRedirect={false} />
+              return 1
+            })
+            .map(({ component: Action, props }, index) => {
+              if (!Action) {
+                return null
+              }
+
+              return <Action key={index} {...props} />
```

Comment:
It's a bad practice to use the array's `index` as a key. Better to assign an unique key to the items